<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MQN1PC7" failures="2" tests="9" name="TwoClassesTest" time="0.316" errors="0" timestamp="2022-11-27T14:52:37 IST">
  <testcase classname="annotations.TestSuit2" name="r11" time="0.037"/>
  <testcase classname="annotations.TestSuit2" name="r12" time="0.023">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at annotations.TestSuit2.r12(TestSuit2.java:15)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- r12 -->
  <testcase classname="annotations.TestSuit2" name="r13" time="0.013"/>
  <testcase classname="annotations.TestSuit2" name="r14" time="0.005"/>
  <testcase classname="annotations.TestSuit2" name="r15" time="0.007"/>
  <testcase classname="annotations.TestSuit2" name="r16" time="0.009"/>
  <testcase classname="annotations.TestSuit2" name="r17" time="0.008">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at annotations.TestSuit2.r17(TestSuit2.java:39)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- r17 -->
  <testcase classname="annotations.TestSuit2" name="r18" time="0.007"/>
  <testcase classname="annotations.TestSuit2" name="r19" time="0.012"/>
</testsuite> <!-- TwoClassesTest -->
